### DISCUSSION TITLE
Unable to make the software work on one computer

### DISCUSSION THREAD

**Discussion by viper377** [valinet/ExplorerPatcher#2621] [Q&A]
Hello,

Windows 11 23H2, build 22631.2861.  Latest non insider update as far as Windows Update is concerned.

My problem is, whenever I install Explorer Patcher, I can not right click anything on the desktop.  The screen flickers as if it was crashing, but then nothing happens.  A moment of... hesitation from Windows, then nothing.

I tried released 22621.2861.62.2 of EP, but this happened on previous versions too (as of 2 weeks ago, this is a new build and I just installed the 23H2 update yesterday night, the crash was there before).

Here's where it gets interesting:  I have EP installed on another Win11 computer, same build, and it's working great.
The machine do not contain the same hardware, one is AM4 based, the other one is AM5.  Both are new Windows install.  Am4 is a refresh, AM5 is a complete new install I fracked my boot.

Only difference between the two is I tried to install EP on a version of Win11 that created conflicts and crashed a lot with EP installed.

I'm still slowly reinstalling my softwares on the AM5 machine, so it is still relatively "clean".  Steam is there, the drivers are there, since EP wasn't working, I installed Start11 from Stardock, but I'll trade the nice look for my auto-hide taskbar on top anyday.

It is similar to that [issue here](https://github.com/valinet/ExplorerPatcher/discussions/2182).

I've tried EP first, alone, nothing else.  I've tried Start11, didn't like it, tried reverting back to a newer version of EP after uninstalling Start11, no joy.  Tried EP with Start11, still no joy.  Uninstalled both, then tried EP alone again, no joy.

My interface language is set to English, keyboard is set to French Canada.

I'm usually not an idiot, but I'm a bit lost and would appreciate any help anyone can give.

Thanks.

Edit:
I forgot to add that both computers use [Display Fusion](https://www.displayfusion.com/) by Binary Software.  Both computers use a single monitor.

**Comment by pyrates999** [comment]
duplicate of: https://github.com/valinet/ExplorerPatcher/issues/2218

please close this
